[2025-05-05T21:37:03.687+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-05T21:36:00+00:00 [queued]>
[2025-05-05T21:37:03.695+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-05T21:36:00+00:00 [queued]>
[2025-05-05T21:37:03.695+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-05T21:37:03.710+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): sync_and_load_mysql> on 2025-05-05 21:36:00+00:00
[2025-05-05T21:37:03.715+0000] {standard_task_runner.py:60} INFO - Started process 4268 to run task
[2025-05-05T21:37:03.719+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mymind_mongo_to_mysql_etl', 'sync_and_load_mysql', 'scheduled__2025-05-05T21:36:00+00:00', '--job-id', '31970', '--raw', '--subdir', 'DAGS_FOLDER/mymind_etl_dag.py', '--cfg-path', '/tmp/tmpe4r7dp_6']
[2025-05-05T21:37:03.720+0000] {standard_task_runner.py:88} INFO - Job 31970: Subtask sync_and_load_mysql
[2025-05-05T21:37:03.772+0000] {task_command.py:423} INFO - Running <TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-05T21:36:00+00:00 [running]> on host 52beca4b988b
[2025-05-05T21:37:03.881+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mymind_mongo_to_mysql_etl' AIRFLOW_CTX_TASK_ID='sync_and_load_mysql' AIRFLOW_CTX_EXECUTION_DATE='2025-05-05T21:36:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-05T21:36:00+00:00'
[2025-05-05T21:37:03.883+0000] {mymind_etl_dag.py:301} INFO - Iniciando sincronización y carga a MySQL: 5 usuarios, 10 transcripciones.
[2025-05-05T21:37:03.893+0000] {base.py:83} INFO - Using connection ID 'mysql_mymind_dw' for task execution.
[2025-05-05T21:37:03.904+0000] {mymind_etl_dag.py:310} INFO - Obteniendo IDs actuales desde MySQL...
[2025-05-05T21:37:03.905+0000] {mymind_etl_dag.py:321} INFO - Usuarios a eliminar: 0
[2025-05-05T21:37:03.906+0000] {mymind_etl_dag.py:322} INFO - Transcripciones a eliminar: 0
[2025-05-05T21:37:03.906+0000] {mymind_etl_dag.py:355} INFO - Cargando 5 usuarios en la tabla users...
[2025-05-05T21:37:03.907+0000] {mymind_etl_dag.py:380} INFO - Usuarios cargados exitosamente.
[2025-05-05T21:37:03.908+0000] {mymind_etl_dag.py:393} INFO - Cargando 10 transcripciones en la tabla transcriptions...
[2025-05-05T21:37:03.910+0000] {mymind_etl_dag.py:423} INFO - Transcripciones cargadas exitosamente.
[2025-05-05T21:37:03.910+0000] {mymind_etl_dag.py:430} INFO - Sincronización y carga a MySQL completada con éxito.
[2025-05-05T21:37:03.911+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-05T21:37:03.922+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mymind_mongo_to_mysql_etl, task_id=sync_and_load_mysql, execution_date=20250505T213600, start_date=20250505T213703, end_date=20250505T213703
[2025-05-05T21:37:03.970+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-05T21:37:04.003+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
